runtime/lua/vim/lsp/completion.lua

```diff
@@ -615,6 +615,12 @@ local function on_complete_done()
   end
 end
 
+---@param bufnr integer
+---@return string
+local function get_completions_augroup_name(bufnr)
+  return string.format('vim/lsp/completion-%d', bufnr)
```

Comment:
we are using the "dot" convention now, `:help dev-namespace-name`
```suggestion
  return string.format('nvim.lsp.completion_%d', bufnr)
```

Reply:
ok, addressed